design-proposals/klip-15-new-api-and-client.md

```diff
@@ -176,23 +174,35 @@ Each JSON object will be separated by a new line.
 
 To terminate the insert stream the client should end the request.
 
-If acks are requested then the response will be written to the response when each row has been
+Acks will be written to the response when each row has been
 successfully committed to the underlying topic. Rows are committed in the order they are provided.
-Each ack in the response is just an empty JSON object, separated by newlines:
+Each ack in the response is a JSON object, separated by newlines:
 
 ````
-{}
-{}
-{}
-{}
+{"status":"ok","seq":0}
+{"status":"ok","seq":2}
+{"status":"ok","seq":1}
+{"status":"ok","seq":3}
 ````
 
+A successful ack will contain a field `status` with value `ok`.
+
+All ack responses also contain a field `seq` with a 64 bit signed integer value. This number
+corresponds to the sequence of the insert on the request. The first send has sequence `0`, the second
+`1`, the third `3`, etc. It allows the client to correlate the ack to the corresponding send.
```

Comment:
Should this be "the third `2`" instead of "the third `3`", because of the 0-indexing?